cypress/e2e/organization_admin/repositories.test.ts

```diff
@@ -80,21 +103,22 @@ describe('ACCEPTANCE: ORGANIZATION_ADMIN | REPOSITORIES', () => {
   // TBD - Could do more testing here. Only cursory testing for
   // presence of field objects. No testing of form behavior yet.
   it('has an add repository page', () => {
-    cy.visit('/providers/datacite/repositories/new');
-    cy.url().should('include', '/providers/datacite/repositories/new').then(() => {
+    cy.visit('/providers/' + provider_id + '/repositories/new');
+    cy.url().should('include', '/providers/' + provider_id + '/repositories/new').then(() => {
 
-      cy.wait(waitTime);
+      cy.wait(waitTime2);
       cy.get('h2.work').contains('DataCite');
-      cy.get('a#account_menu_link').should('contain', 'DATACITE');
+      cy.get('a#account_menu_link').should('contain', Cypress.env('organization_admin_username').toUpperCase());
 
       cy.get('h3.edit').contains(/Add\s*Repository/);
 
       cy.get('#repository-id').should('be.visible');
       cy.get('#client-type').should('be.visible');
       cy.get('#client-type .ember-power-select-selected-item').should('contain', 'Repository');
 
+      /* - TEMPORARILY SKIP THIS
```

Comment:
How temporary.  Is there a timeline to revisit this?

Reply:
Yes.  The timeline is the next Fabrica upgrade to Ember 3.28.  I left a provision under the heading of 'Improving Cypress Testing' to address the flakiness in tests that fill out forms.